pages/articles.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+    <div class="pa-5">
+        <div v-if="loading">
+            <Loading :active="loading"
+                :can-cancel="false"
+                :is-full-page="true"
+                color="#007bff"
+            />
+        </div>
+        <div v-else>
+            <v-row align="center" class="mx-0">
+                <v-col :cols="4" v-for="article, index in currentArticleList" :key="index">
```

Comment:
if you pass a string value to cols prop it would also work,
so basically we prefer to pass string instead of number that requires `:` (binding)

Reply:
Fixed.